Gemfile

```diff
@@ -121,8 +121,7 @@ gem 'unicorn'
 gem 'i18n', '>=0.6.8'
 # see https://groups.google.com/forum/#!topic/ruby-security-ann/pLrh6DUw998
 
-gem 'nokogiri', '>=1.5.11'
-# see https://groups.google.com/forum/#!topic/ruby-security-ann/DeJpjTAg1FA
+gem 'nokogiri'
```

Comment:
Because people tend to use "bundle update" instead of "bundle (install)", we tend to restrict the versions.